client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/useInteractionsTab.ts

```diff
@@ -16,20 +18,32 @@ const symptomaticInvestigationDaysBeforeConfirmed: number = 10;
 const useInteractionsTab = (props: useInteractionsTabInput) :  useInteractionsTabOutcome => {
     const {parseAddress} = useGoogleApiAutocomplete();
     const { interactions, setInteractions } = props;
+
     const epidemiologyNumber = useSelector<StoreStateType, number>(state => state.investigation.epidemiologyNumber);
 
-    const getDatesToInvestigate = (startInvestigationDateVariables: StartInvestigationDateVariables) : Date[] => {
-        const { hasSymptoms, symptomsStartDate, endInvestigationDate } = startInvestigationDateVariables;
-        if (!endInvestigationDate) return [];
-        let startInvestigationDate : Date;
-        const symptomaticStartInvestigationDate =
-            subDays(endInvestigationDate, symptomaticInvestigationDaysBeforeConfirmed);
-        if (hasSymptoms) {
-            if (symptomsStartDate) startInvestigationDate = max([subDays(symptomsStartDate, investigationDaysBeforeSymptoms), symptomaticStartInvestigationDate]);
-            else startInvestigationDate = subDays(endInvestigationDate, symptomaticInvestigationDaysBeforeConfirmed);
+    const getCoronaTestDate = (setTestDate: React.Dispatch<React.SetStateAction<Date | null>>) => {
+        axios.get('/clinicalDetails/coronaTestDate').then((res: any) => {
+            if(res.data !== null) {
+                setTestDate(convertDate(res.data));
+            }
+        })
+    }
+
+    const getDatesToInvestigate = (clinicalDetailsCtxt: ClinicalDetailsDataAndSet, coronaTestDate: Date | null) : Date[] => {
+        const { doesHaveSymptoms, symptomsStartDate } = clinicalDetailsCtxt.clinicalDetailsData;
+        if(coronaTestDate !== null) {
```

Comment:
Did you checked what is the max time to investigate? Because if the date to start investigate when there is symptoms start date is earlier then 7 days pre test, do we investigate more than 7 days?

Reply:
i said in the group that tal should get back to me on that